cirq/ops/raw_types.py

```diff
@@ -262,21 +262,14 @@ def __pow__(self, power):
     def __call__(self, *args, **kwargs):
         return self.on(*args, **kwargs)
 
-    def controlled_by(self,
-                      *control_qubits: Qid,
-                      control_values: Optional[Sequence[
-                          Union[int, Collection[int]]]] = None) -> 'Gate':
+    def control(self,
+                control_values: Optional[Sequence[
+                    Union[int, Collection[int]]]] = None) -> 'Gate':
         """Returns a controlled version of this gate.
-
-        Args:
-            control_qubits: Optional qubits to control the gate by.
```

Comment:
Document the arguments.  (It looks like I forgot to document control_values.)

Reply:
Documented. Also added documentation to the Operation controlled_by and TODOs for adding these arguments to that and the control protocol.